<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Mnemosyne</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mnemosyne.qrc" >
    <normaloff>:/mnemosyne/pixmaps/mnemosyne.png</normaloff>:/mnemosyne/pixmaps/mnemosyne.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew" />
    <addaction name="actionFileOpen" />
    <addaction name="actionFileSave" />
    <addaction name="actionFileSaveAs" />
    <addaction name="separator" />
    <addaction name="actionImport" />
    <addaction name="actionExport" />
    <addaction name="separator" />
    <addaction name="actionFileExit" />
   </widget>
   <widget class="QMenu" name="menu_Deck" >
    <property name="title" >
     <string>&amp;Deck</string>
    </property>
    <addaction name="actionAddCards" />
    <addaction name="actionEditCurrentCard" />
    <addaction name="actionDeleteCurrentCard" />
    <addaction name="separator" />
    <addaction name="actionEditDeck" />
    <addaction name="actionActivateCategories" />
    <addaction name="actionCleanDuplicates" />
    <addaction name="separator" />
    <addaction name="actionShowStatistics" />
   </widget>
   <widget class="QMenu" name="menu_Settings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionShowToolbar" />
    <addaction name="actionConfiguration" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionGettingStarted" />
    <addaction name="actionTip" />
    <addaction name="separator" />
    <addaction name="actionHelpAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Deck" />
   <addaction name="menu_Settings" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddCards" />
   <addaction name="actionEditCurrentCard" />
   <addaction name="actionDeleteCurrentCard" />
   <addaction name="actionEditDeck" />
   <addaction name="actionShowStatistics" />
  </widget>
  <action name="actionActivateCategories" >
   <property name="text" >
    <string>Activate cate&amp;gories</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAddCards" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/add.png</normaloff>:/mnemosyne/pixmaps/add.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Add cards</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCleanDuplicates" >
   <property name="text" >
    <string>&amp;Clean duplicates</string>
   </property>
  </action>
  <action name="actionConfiguration" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/configure.png</normaloff>:/mnemosyne/pixmaps/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure Mnemosyne</string>
   </property>
  </action>
  <action name="actionDefault" >
   <property name="text" >
    <string>default</string>
   </property>
  </action>
  <action name="actionDeleteCurrentCard" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/delete.png</normaloff>:/mnemosyne/pixmaps/delete.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Delete current card</string>
   </property>
   <property name="shortcut" >
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionEditCurrentCard" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/edit.png</normaloff>:/mnemosyne/pixmaps/edit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Edit current card</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEditDeck" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/deck.png</normaloff>:/mnemosyne/pixmaps/deck.png</iconset>
   </property>
   <property name="text" >
    <string>Edit &amp;deck</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExport" >
   <property name="text" >
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionFileExit" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/exit.png</normaloff>:/mnemosyne/pixmaps/exit.png</iconset>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionFileNew" >
   <property name="icon" >
    <iconset>
     <normaloff>../../pixmaps/filenew.png</normaloff>../../pixmaps/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpen" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/fileopen.png</normaloff>:/mnemosyne/pixmaps/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/filesave.png</normaloff>:/mnemosyne/pixmaps/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/filesaveas.png</normaloff>:/mnemosyne/pixmaps/filesaveas.png</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionImport" >
   <property name="text" >
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionPressEnter" >
   <property name="text" >
    <string>pressEnter</string>
   </property>
   <property name="shortcut" >
    <string>Enter</string>
   </property>
  </action>
  <action name="actionPressR" >
   <property name="text" >
    <string>pressR</string>
   </property>
   <property name="shortcut" >
    <string>R</string>
   </property>
  </action>
  <action name="actionPressReturn" >
   <property name="text" >
    <string>pressReturn</string>
   </property>
   <property name="shortcut" >
    <string>Return</string>
   </property>
  </action>
  <action name="actionPressSpace" >
   <property name="text" >
    <string>pressSpace</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="actionGettingStarted" >
   <property name="text" >
    <string>&amp;Getting started</string>
   </property>
  </action>
  <action name="actionShowStatistics" >
   <property name="icon" >
    <iconset resource="mnemosyne.qrc" >
     <normaloff>:/mnemosyne/pixmaps/statistics.png</normaloff>:/mnemosyne/pixmaps/statistics.png</iconset>
   </property>
   <property name="text" >
    <string>Show s&amp;tatistics</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionShowToolbar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show &amp;toolbar</string>
   </property>
  </action>
  <action name="actionTip" >
   <property name="text" >
    <string>&amp;Tip of the day</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mnemosyne.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionAddCards</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>addCards</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>164</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
